build-support/bin/copy_to_s3.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import annotations
+
+import argparse
+import logging
+import os
+import shutil
+import subprocess
+from pathlib import PurePath
+
+from common import die
+
+logger = logging.getLogger(__name__)
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--src-prefix", required=True, help="A subdirectory of the cwd to copy files from."
+    )
+    parser.add_argument("--dst-prefix", required=True, help="An s3 URL to copy to.")
+    parser.add_argument(
+        "--path",
+        default="",
+        help=(
+            "A subdirectory of --src-prefix to copy to the same relative path under --dst-prefix. "
+            "That is, src_prefix/path will be copied to dst_prefix/path. If unspecified, the "
+            "entire src_prefix will be copied."
+        ),
+    )
+    parser.add_argument(
+        "--region",
+        help="The AWS region to connect to.",
+        default="us-east-1",
+    )
+    options = parser.parse_args()
+    perform_copy(
+        src_prefix=options.src_prefix,
+        dst_prefix=options.dst_prefix,
+        path=options.path,
+        dst_region=options.dst_region,
+    )
+
+
+def _run(args, env=None) -> None:
+    try:
+        subprocess.run(
+            args=args,
+            env=env,
+            capture_output=True,
+            check=True,
+        )
+    except subprocess.CalledProcessError as ex:
+        logger.error(f"Process `{' '.join(args)}` failed with exit code {ex.returncode}.")
+        logger.error(f"stdout:\n{ex.stdout.decode()}\n")
```

Comment:
You could `text=True` the run call and save a bit of hoops here.

Reply:
I just copied that from its previous location unchanged.